editor/editor_file_system.h

```diff
@@ -283,6 +283,7 @@ class EditorFileSystem : public Node {
 		const ImportFile *reimport_files;
 		int reimport_from;
 		int max_index = 0;
+		Mutex max_index_lock;
```

Comment:
```suggestion
		Mutex max_index_mutex;
```
I'd name it `mutex` to avoid any confusion